moveit_ros/move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -64,20 +64,26 @@ void move_group::MoveGroupMoveAction::executeMoveCallback(const moveit_msgs::Mov
   context_->planning_scene_monitor_->waitForCurrentRobotState(ros::Time::now());
   context_->planning_scene_monitor_->updateFrameTransforms();
 
+  planning_scene_monitor::LockedPlanningSceneRO lscene(context_->planning_scene_monitor_);
+  const robot_state::RobotState& current_state = lscene->getCurrentState();
+  moveit_msgs::MoveGroupGoal new_goal = *goal;
+  robot_state::robotStateToRobotStateMsg(current_state, new_goal.request.start_state);
+  moveit_msgs::MoveGroupGoalConstPtr goal_(&new_goal);
```

Comment:
this seems dangerous that the class member variable's memory will be unloaded when this function ends, because ``new_goal`` is a local variable. are you sure this is safe?

Reply:
actually, the underscore in ``goal_`` is misused as it represents a member variable when you're using it as a local variable

Reply:
You were right about scope issue. It has an allocated memory now. Misused name is changed. Thank you very much :)